receiver/zookeeperreceiver/metadata.yaml

```diff
@@ -76,4 +76,25 @@ metrics:
     description: Maximum number of file descriptors that a ZooKeeper server can open.
     unit: 1
     data:
-      type: int gauge
\ No newline at end of file
+      type: int gauge
+  zookeeper.packets.received:
+    description: Number of ZooKeeper packets received by a server.
+    unit: 1
+    data:
+      type: int sum
+      monotonic: true
+      aggregation: cumulative
+  zookeeper.packets.sent:
+    description: Number of ZooKeeper packets sent by a server.
+    unit: 1
+    data:
+      type: int sum
+      monotonic: true
+      aggregation: cumulative
+  zookeeper.fsync_threshold_exceeds:
+    description: Number of times fsync duration has exceeded warning threshold.
+    unit: 1
+    data:
+      type: int sum
+      monotonic: true
+      aggregation: cumulative
```

Comment:
new line?

Reply:
@asuresh4 please send a different PR for this.

Reply:
👍 will do.